tools/packaging/release/release-notes.sh

```diff
@@ -94,6 +104,17 @@ EOF
 ## ${repo} Changes
 $(changes)
 
+## Survey
+
+If you would like to help the Kata Containers community understand:
```

Comment:
```suggestion
Please help the Kata Containers community understand:
```

Reply:
To make the call-to-action more active.

Reply:
Thanks, @fidencio - I've actually tweaked it a bit further and swapped the order around a bit. wdyt?

Reply:
@ildikov - Apologies, that comment was meant for you :smile:

Reply:
@jodh-intel :) I like your new version of the text, thank you for tweaking it!